sklearn/tests/test_docstring_parameters.py

```diff
@@ -288,8 +288,7 @@ def test_fit_docstring_attributes(name, Estimator):
         with ignore_warnings(category=FutureWarning):
             assert hasattr(est, attr.name)
 
-    IGNORED = {'Birch', 'LarsCV', 'Lasso',
-               'OrthogonalMatchingPursuit'}
+    IGNORED = {}
```

Comment:
I would just remove this dictionary and the following `if` statement, because they are not used anymore.

Reply:
@TomDLT I'll add another commit with the needed changes.